xbmc/guilib/GUIWindow.cpp

```diff
@@ -618,6 +618,15 @@ bool CGUIWindow::OnMessage(CGUIMessage& message)
       if (HasID(message.GetSenderId()))
       {
         m_focusedControl = message.GetControlId();
+        // We ensure that all others childrens have focus disabled,
```

Comment:
related to this, the use case is simple:
move the subtitle bar inside/over the kodi logo box (ratio control),
then move the mouse pointer -before- over the logo then -after- over the subtitle bar
will result that the ratio control and the subtitle bar have both the focus

Reply:
I ask this because I've found this problem earlier in other contexts, eg when you have two button too close to each other, kodi will focus both. This is easily reproduced in those dialogs where you have the "ok" and "cancel" buttons on the right of the dialog.

![image](https://user-images.githubusercontent.com/7375276/148360379-1358e3c8-e973-4168-b8af-53a4f307bd7c.png)

Reply:
I didn't notice that, but i had seen this:
![immagine](https://user-images.githubusercontent.com/3257156/148361082-e954168b-e226-4db4-94cc-a26c1fbd3f1f.png)
i think should same problem, but this change not fix these cases, maybe because are within a list control?